README.md

```diff
@@ -86,6 +86,7 @@ A ferret is actually a really good spirit animal for this project: cute, small,
 - [psycopg3]
 - [pip]
 - [packaging]
+- [feincms3]
 
 I'm also a maintainer on the last two, so... that covers responsible disclosure. :)
```

Comment:
this line is now wrong :)

Reply:
Oh right :-) Thanks! Good catch.

I'm not sure whether @pradyunsg wants to start advertising other projects using Furo, so maybe it's not worth it fixing this.